apps/sciterminal/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+APP_NAME="sciterminal"
+APP_VERSION="20241018"
```

Comment:
that's not used. However I'd expose some versions like:

- plink2
- shapeit5
- bcftools